content/issues/2023-07-08-flexget.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 'New Container: FlexGet'
+date: '2023-07-09 12:00:00Z'
```

Comment:
Does this date need to match the file name?

Reply:
It doesn't *need* to, but I've updated both dates to match